docs/trace.md

```diff
@@ -191,7 +259,7 @@ The meaning of the trace timestamps is:
 
   * Compute: The time the inference request spent executing the actual
```

Comment:
Should be at same level as "Queue" above

Reply:
"Compute" is the same level as "Queue". What's your meaning?